code/forms/DMSDocumentRequestForm.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class DMSDocumentRequestForm extends Form
+{
+    public function __construct(Controller $controller, $name, FieldList $fields, FieldList $actions, $validator = null)
+    {
+        $requiredFields = array(
+            'ReceiverName',
+            'ReceiverPhone',
+            'ReceiverEmail',
+            'DeliveryAddressLine1',
+            'DeliveryAddressCountry',
+            'DeliveryAddressPostCode',
+        );
+        foreach ($fields as $field) {
+            if (in_array($field->name, $requiredFields)) {
+                $field->addExtraClass('requiredField');
+            }
+        }
+        $validator = new RequiredFields($requiredFields);
+
+        $actions->push(new FormAction('doRequestSend', 'Send your request'));
+
+        parent::__construct($controller, $name, $fields, $actions, $validator);
+    }
+
+    public function forTemplate()
+    {
+        return $this->renderWith(array($this->class));
+    }
+}
```

Comment:
Is the purpose of this class just to define the required fields? If so you could just use `Form::create(...)->setValidator($validator);` from the controller and remove the need for the new class?